### PR TITLE
Retry on ssl.SSLError in vm_util - this resolves test flakiness

### PR DESCRIPTION
**Pull request by ankushagarwal**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/987)

### CODE REVIEW

File: testing/vm_util.py [hunk 1]
```diff
@@ -37,13 +38,15 @@ def wait_for_operation(client,
...
-
+    try:
+      if zone:
+        op = client.zoneOperations().get(
+          project=project, zone=zone, operation=op_id).execute()
+      else:
+        op = client.globalOperations().get(project=project,
+                                           operation=op_id).execute()
+    except ssl.SSLError as e:
+      logging.error("Ignoring error %s", e)
```
Comment (lluunn) [inline testing/vm_util.py]: should continue?

Comment (ankushagarwal) [reply testing/vm_util.py]: Done

**Pull request merged**